gifDictionary.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+def main():
+    result = requests.get("https://api.giphy.com/v1/gifs/trending?api_key=frUdMjsYJxj0gyHTP94XuDOnVodkPuY0&limit=25&offset=0&rating=g")
```

Comment:
This PR will permanently add your API key to your git repository. This is like adding a password to your repository. This is a problem. You have two options:

(1) Remove this pull request/branch altogether and create a new one that doesn't have the API key before you merge it.

(2) Go ahead and merge it then read the documentation on how to [permanently remove secrets from a github repo](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository).